Samples.xcodeproj/project.pbxproj

```diff
@@ -190,6 +192,7 @@
 			dstPath = "";
 			dstSubfolderSpec = 7;
 			files = (
+				D7E9EF292A1D2219000C4865 /* SetMinAndMaxScaleView.swift in Copy Source Code Files */,
```

Comment:
I'll let @yo1995 comment on this but it looks like most of the files in this list are alphabetized, should this be as well?

Reply:
<img width="1001" alt="Screenshot 2023-05-23 at 11 33 10" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/16397058/58d923d7-9bea-4fee-8a09-88eda6629582">

Reply:
Dang! I wasn't paying attention to the order of this list until now. 🤦 
It seems Xcode doesn't have a way to sort it in GUI: https://stackoverflow.com/questions/24326147 

However, I don't think the order matters too much for a few reasons

- Xcode doesn't allow us to do so automatically 🤪 and it's error-prone to change order manually
- The source code file names don't strictly follows the sample name, so if sorted alphabetically, it is possible that the swift files for 1 sample is scattered around multiple places
- the order doesn't matter. We only need a "unordered set" of filenames so they are copied to the bundle.
- (a small benefit) Adding the files using the default order preserves the order of when they were added, though this info is not important at all.

so no need to worry about the order.